test/unit/web-runtime/next-response-cookies.test.ts

```diff
@@ -27,9 +26,17 @@ it('reflect .set into `set-cookie`', async () => {
     value: 'barz',
   })
 
-  expect(Object.fromEntries(response.headers.entries())['set-cookie']).toBe(
+  expect(response.headers.get('set-cookie')).toBe(
```

Comment:
Expected due to https://github.com/vercel/edge-runtime/pull/942. I added an additional assertion to check that `entries()` still contains all values.